parser-typechecker/tests/Unison/Test/TermParser.hs

```diff
@@ -52,6 +52,8 @@ test1 = scope "termparser" . tests . map parses $
   , "\"forty two\""
   , "[1,2,3]"
   , "\"abc\""
+  , "$x"
+  , "$\\n"
```

Comment:
What does the `$\\n` parse as? Do you need both `\\`? Why isn't it just `$\n` ?

Is the `$` character written `$$`?

__Edit:__ disregard, I get it. `$\` is the literal character `\`. `$\\n` is the newline character. You need some way of distinguishing the two.

Reply:
The extra backslash here is escaping in the haskell string; the literal for a newline in unison would be `$\n`. A literal backslash would be `$\\`, since `\` itself needs to be escaped. `$` doesn't, so yes the literal character would be `$$`.